src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/types/RecordType.kt

```diff
@@ -3,4 +3,5 @@ package uk.gov.justice.digital.hmpps.personrecord.model.types
 enum class RecordType {
   PRIMARY,
   PREVIOUS,
+  HISTORIC,
```

Comment:
can the PrisonSexualOrientationEntity have all of these values?
can the AddressEntity?
If so 👍